src/store.rs

```diff
@@ -51,13 +64,13 @@ impl Store {
             "url" => SafeDisplay(postgres_url.as_str())
         );
 
-        // Create the store
-        let store = Store {
+        // Create and return the store
+        Store {
             logger: logger.clone(),
             conn: pool,
-        };
-        // Return the store
-        store
+            schema_cache: Mutex::new(LruCache::with_capacity(100)),
```

Comment:
Did you chose this magic number or is this what they use as well?

Reply:
It is what they had.